Use 'ActivityFailed' exception instead of 'FailedActivity' in chaosk8s/pods

NithyaNatarajan opened the pull request:
Have refactored `chaosk8s/pods/actions.py` and  `chaosk8s/pods/pods.py` to use `chaoslib.exceptions.ActivityFailed` instead of `chaoslib.exceptions.FailedActivity`

Signed-off-by: Nithya Natarajan <nithyanatarajn@gmail.com>

Lawouach commented:
Great thanks!

Could you bump the requirements to expect chaostoolkit-lib>=0.20.0 when that exception renaming occured please? Thanks :)

https://github.com/chaostoolkit/chaostoolkit-lib/blob/master/CHANGELOG.md

NithyaNatarajan commented:
@Lawouach I have updated requirements.txt to use chaostoolkit-lib>=0.20.0 and I have also removed all occurrences of `FailedActivity` from the codebase. Kindly have a look.

Lawouach commented on chaosk8s/actions.py:
```diff
@@ -1,18 +1,16 @@
-from chaoslib.exceptions import FailedActivity
-from chaoslib.types import MicroservicesStatus, Secrets
-from logzero import logger
+import yaml
```
We favour import sorted alaphabetically :)
So that one should remain the last one in this list.
